unlock-app/package.json

```diff
@@ -74,6 +76,7 @@
     "remark-html": "16.0.1",
     "remark-parse": "11.0.0",
     "remove-markdown": "0.5.0",
+    "sequelize": "6.37.3",
```

Comment:
I don't think it is good practice to use a backend library in our front-end app~